main.go

```diff
@@ -70,19 +74,19 @@ func main() {
 			continue
 		}
 		for _, user := range userList {
-			// This user has left message page, ready to receive message.
+			// This user has left message page, ready to receive email and text.
 			// He has new unknown messages
 			if (time.Now().Unix()-int64(user.LastGetNewMessageTime)) > globCfg.TimeLimit &&
 				(user.LastGetNewMessageTime >= user.LastSendEmailTime ||
-					int(time.Now().Unix()) - user.LastSendEmailTime > 24 * 60 * 60 ) {
+					int(time.Now().Unix())-user.LastSendEmailTime > int(globCfg.EmailLockTime)*60) {
 
 				var s = "User [%d] has read his message and left message page, "
 				if user.LastGetNewMessageTime >= user.LastSendEmailTime {
 					s += "he has never received a notification since last visit message page, "
 				} else {
 					s += "he received a notification last time one day ago, "
```

Comment:
This `one day` ago could be confusing since it is configurable and may not be one day